packages/sdk/src/types/runtime/assets/assetContainer.ts

```diff
@@ -188,6 +188,8 @@ export class AssetContainer {
 		if (reply.failureMessage || reply.assets.length !== 1) {
 			throw new Error(`Creation/Loading of asset ${asset.name} failed: ${reply.failureMessage}`);
```

Comment:
Unrelated feedback, but this shouldn't be throwing an exception. Log an error instead.

Reply:
This error bubbles up to rejecting the `Asset.created` promise. Isn't this what it's for?

---
In reply to: [315937817](https://github.com/microsoft/mixed-reality-extension-sdk/pull/421#discussion_r315937817) [](ancestors = 315937817)